docs/arcadia-users-group/20240416-intro-to-ml1/make_peptides_ml_dataset.md

```diff
@@ -0,0 +1,264 @@
+## Documentation for how the dataset was made
+
+0. We used the environment defined in `make_peptides_ml_dataset.yml` to make the data set below.
+1. Download the peptipedia database:  https://drive.google.com/file/d/1x3yHNl8k5teHlBI2FMgl966o51s0T8i_/view?usp=sharing.
```

Comment:
Getting a 400 on that link:

![image](https://github.com/Arcadia-Science/arcadia-computational-training/assets/1061230/0c030ae5-7cde-425b-9d9f-22099cb5f051)

Also, what is the original source of the data?